--- bitlbee-3.5.1/ipc.c.orig	Mo. Jan 30 21:44:32 2017
+++ bitlbee-3.5.1/ipc.c	Sa. Aug 18 20:56:04 2018
@@ -181,7 +181,8 @@
 
 	if (child->to_child == NULL ||
 	    g_slist_find(child_list, child->to_child) == NULL) {
-		return ipc_master_takeover_fail(child, FALSE);
+		ipc_master_takeover_fail(child, FALSE);
+		return;
 	}
 
 	if (strcmp(cmd[1], "AUTH") == 0) {
@@ -201,7 +202,8 @@
 			}
 			g_free(fwd);
 		} else {
-			return ipc_master_takeover_fail(child, TRUE);
+			ipc_master_takeover_fail(child, TRUE);
+			return;
 		}
 	} else if (strcmp(cmd[1], "DONE") == 0 || strcmp(cmd[1], "FAIL") == 0) {
 		/* Old connection -> Master */
