--- freeciv-2.3.0/client/climisc.c	Mon Aug  8 06:20:48 2011
+++ freeciv-2.3.0/client/climisc.c	Fri Oct 21 10:52:04 2011
@@ -315,7 +315,7 @@
 **************************************************************************/
 void global_warming_scaled(int *chance, int *rate, int max)
 {
-  return catastrophe_scaled(chance, rate, max,
+  catastrophe_scaled(chance, rate, max,
                             game.info.heating, game.info.globalwarming,
                             game.info.warminglevel);
 }
@@ -325,7 +325,7 @@
 **************************************************************************/
 void nuclear_winter_scaled(int *chance, int *rate, int max)
 {
-  return catastrophe_scaled(chance, rate, max,
+  catastrophe_scaled(chance, rate, max,
                             game.info.cooling, game.info.nuclearwinter,
                             game.info.coolinglevel);
 }
--- freeciv-2.4.0/server/scripting/api_server_edit.c	2013-10-28 10:23:15.194900152 +0100
+++ freeciv-2.4.0/server/scripting/api_server_edit.c	2013-10-28 10:23:23.641127154 +0100
@@ -69,7 +69,7 @@
   LUASCRIPT_CHECK(L, 0 < num_role_units(L_PARTISAN),
                   "no partisans in ruleset");
 
-  return place_partisans(ptile, pplayer, count, sq_radius);
+  place_partisans(ptile, pplayer, count, sq_radius);
 }
 
 /*****************************************************************************
