--- gcc-4.8.4/gcc/config/sol2.h.orig	2015-02-07 16:25:45.417613612 +0100
+++ gcc-4.8.4/gcc/config/sol2.h	2015-02-27 12:08:11.579484015 +0100
@@ -120,12 +120,17 @@
 
 #ifndef CROSS_DIRECTORY_STRUCTURE
 #undef MD_EXEC_PREFIX
-#define MD_EXEC_PREFIX "/usr/ccs/bin/"
+#define MD_EXEC_PREFIX "/usr/bin/"
 
 #undef MD_STARTFILE_PREFIX
-#define MD_STARTFILE_PREFIX "/usr/ccs/lib/"
+#define MD_STARTFILE_PREFIX "/usr/lib/"
 #endif
 
+/* #undef LINK_LIBGCC_SPECIAL_1 */
+/* #define LINK_LIBGCC_SPECIAL_1=1 */
+#undef LINK_LIBGCC_SPEC
+#define LINK_LIBGCC_SPEC "%{m64:" "-R /usr/gcc/4.8/lib/"ARCH64_SUBDIR":/usr/gcc/lib/"ARCH64_SUBDIR" %D" "}%{!m64:" "-R /usr/gcc/4.8/lib:/usr/gcc/lib %D" "}"
+
 #undef STARTFILE_ARCH32_SPEC
 #define STARTFILE_ARCH32_SPEC "%{ansi:values-Xc.o%s} \
 			    %{!ansi:values-Xa.o%s}"
