--- harfbuzz-1.0.6/util/ansi-print.cc.orig	2015-05-19 03:37:46.066389686 +0200
+++ harfbuzz-1.0.6/util/ansi-print.cc	2016-04-23 15:14:25.119903244 +0200
@@ -298,7 +298,7 @@
     }
     if (best_s < score) {
       static const char *lower[7] = {"▁", "▂", "▃", "▄", "▅", "▆", "▇"};
-      unsigned int which = lround (((best_i + 1) * 8) / bi.height);
+      unsigned int which = lround ((double) ((best_i + 1) * 8) / bi.height);
       if (1 <= which && which <= 7) {
 	score = best_s;
 	*inverse = best_inv;
@@ -330,7 +330,7 @@
     }
     if (best_s < score) {
       static const char *left [7] = {"▏", "▎", "▍", "▌", "▋", "▊", "▉"};
-      unsigned int which = lround (((best_i + 1) * 8) / bi.width);
+      unsigned int which = lround ((double) ((best_i + 1) * 8) / bi.width);
       if (1 <= which && which <= 7) {
 	score = best_s;
 	*inverse = best_inv;
