--- src/tools/hunspell.cxx.orig	Mon Jun  2 07:35:50 2014
+++ src/tools/hunspell.cxx	Thu Sep  3 09:50:35 2015
@@ -67,11 +67,15 @@
 #include "xmlparser.hxx"
 #include "odfparser.hxx"
 
+#ifdef __sun
+#define LIBDIR "/usr/share/spell/myspell"
+#else
 #define LIBDIR \
     "/usr/share/hunspell:" \
     "/usr/share/myspell:" \
     "/usr/share/myspell/dicts:" \
     "/Library/Spelling"
+#endif
 #define USEROOODIR \
     ".openoffice.org/3/user/wordbook:" \
     ".openoffice.org2/user/wordbook:" \
