--- irker-2.18/irkerd	2018-12-23 22:12:37.446816789 +0100
+++ irker-2.18/irkerd	2018-12-23 22:15:21.037029765 +0100
@@ -53,6 +53,7 @@
 import json
 import os
 import os.path
+import platform
 try:  # Python 3
     import queue
 except ImportError:  # Python 2
@@ -999,7 +1000,7 @@
         help='message for --immediate mode')
     args = parser.parse_args()
 
-    if not args.log_file and in_background():
+    if not args.log_file and in_background() and platform.system != 'SunOS':
         # The Linux, Mac, and FreeBSD values of the logging device.
         logdev = [x for x in ('/dev/log', '/var/run/syslog', '/var/run/log')
                   if os.path.exists(x) and not os.path.isdir(x)]
