--- kdebase-3.5.10/startkde.orig	2012-11-27 23:33:52.253170083 +0000
+++ kdebase-3.5.10/startkde	2012-11-27 23:41:38.270981643 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/bash
 #
 #  DEFAULT KDE STARTUP SCRIPT ( KDE-3.5.10 )
 #
@@ -7,6 +7,9 @@
 # because we still need to do some cleanup.
 trap 'echo GOT SIGHUP' HUP
 
+# Set PATH for Solaris-based platforms
+PATH=${PATH}:/usr/sfw/bin:/usr/openwin/bin:/usr/X11/bin
+
 # Check if a KDE session already is running
 if kcheckrunning >/dev/null 2>&1; then
 	echo "KDE seems to be already running on this display."
@@ -377,7 +380,7 @@
 fi
 
 # wait if there's any crashhandler shown
-while dcop | grep -q ^drkonqi- ; do
+while dcop | ggrep -q ^drkonqi- ; do
     sleep 5
 done
 
