--- kdelibs-3.5.10/Makefile.in.orig	2008-08-20 17:05:55.000000000 +0100
+++ kdelibs-3.5.10/Makefile.in	2012-11-27 23:58:39.686010197 +0000
@@ -1056,10 +1056,12 @@
 uninstall-apidox:
 	@if test "$(subdir)" != "."; then \
 		if test -d $(DESTDIR)$(kde_htmldir)/en/$(PACKAGE)-apidocs/$(subdir); then \
+		        echo "Cleaning up $(DESTDIR)$(kde_htmldir)/en/$(PACKAGE)-apidocs/$(subdir)"; \	
 			rm -rf $(DESTDIR)$(kde_htmldir)/en/$(PACKAGE)-apidocs/$(subdir); \
 		fi \
 	else \
 		if test -d $(DESTDIR)$(kde_htmldir)/en/$(PACKAGE)-apidocs; then \
+			echo "Cleaning up $(DESTDIR)$(kde_htmldir)/en/$(PACKAGE)-apidocs"; \
 			rm -rf $(DESTDIR)$(kde_htmldir)/en/$(PACKAGE)-apidocs; \
 		fi \
 	fi
