--- libcdio-0.94/include/cdio/types.h.orig   2017-02-01 22:25:23.932114117 +0100
+++ libcdio-0.94/include/cdio/types.h        2017-02-01 22:26:34.502647696 +0100
@@ -58,6 +58,13 @@ typedef uint8_t ubyte;
 #include <unistd.h>
 #endif
 
+#if (defined(sun) && defined(__SVR4))
+  typedef uint8_t u_int8_t;
+  typedef uint16_t u_int16_t;
+  typedef uint32_t u_int32_t;
+  typedef uint64_t u_int64_t;
+#endif
+
   /* default HP/UX macros are broken */
 #if defined(__hpux__)
 # undef UINT16_C

