--- src/lib/MSPUBMetaData.cpp.orig	2014-12-30 21:42:44.000000000 +1000
+++ src/lib/MSPUBMetaData.cpp	2015-06-15 13:54:03.528440435 +1000
@@ -194,7 +194,7 @@
   uint32_t firstDirSectorLocation = readU32(input);
 
   // Seek to the Root Directory Entry
-  size_t sectorSize = pow(2, sectorShift);
+  size_t sectorSize = std::pow(2, sectorShift);
   input->seek((firstDirSectorLocation + 1) * sectorSize, librevenge::RVNG_SEEK_SET);
   // DirectoryEntryName: 64 bytes
   // DirectoryEntryNameLength: 2 bytes
