--- libreoffice-4.4.4.2/configure.ac.orig	2015-06-11 03:09:02.000000000 +1000
+++ libreoffice-4.4.4.2/configure.ac	2015-06-24 14:29:18.485276281 +1000
@@ -7781,7 +7781,7 @@
         # Unset variables set by the above AM_PATH_PYTHON so that
         # we actually do check anew.
         unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
-        AM_PATH_PYTHON([3.3],, [:])
+        AM_PATH_PYTHON([2.6],, [:])
         if test "$PYTHON" = ":"; then
             if test -z "$PYTHON_FOR_BUILD"; then
                 AC_MSG_RESULT([fully internal])
@@ -7835,7 +7835,7 @@
         # we actually do check anew.
         unset PYTHON am_cv_pathless_PYTHON ac_cv_path_PYTHON am_cv_python_version am_cv_python_platform am_cv_python_pythondir am_cv_python_pyexecdir
         # This causes an error if no python command is found
-        AM_PATH_PYTHON([3.3])
+        AM_PATH_PYTHON([2.6])
         python_include=`$PYTHON -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('INCLUDEPY'));"`
         python_version=`$PYTHON -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('VERSION'));"`
         python_libs=`$PYTHON -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('LIBS'));"`
@@ -7906,10 +7906,10 @@
 
 int main(int argc, char **argv) {
        if ((PY_MAJOR_VERSION == 2 && PY_MINOR_VERSION >= 6) ||
-           (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 3)) return 0;
+           (PY_MAJOR_VERSION == 2 && PY_MINOR_VERSION >= 6)) return 0;
        else return 1;
 }
-           ]])],[AC_MSG_RESULT([ok])],[AC_MSG_ERROR([Python >= 3.3 is needed when building with Python 3, or Python >= 2.6 when building with Python 2])],[])
+           ]])],[AC_MSG_RESULT([ok])],[AC_MSG_ERROR([Python >= 2.7 is needed when building with Python 3, or Python >= 2.6 when building with Python 2])],[])
         CFLAGS=$save_CFLAGS
         AC_LANG_POP(C)
 
@@ -7919,8 +7919,8 @@
 
 internal)
     SYSTEM_PYTHON=
-    PYTHON_VERSION_MAJOR=3
-    PYTHON_VERSION_MINOR=3
+    PYTHON_VERSION_MAJOR=2
+    PYTHON_VERSION_MINOR=6
     PYTHON_VERSION=${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}.3
     BUILD_TYPE="$BUILD_TYPE PYTHON"
     # Embedded Python dies without Home set
