--- src/network.c.orig	2017-11-17 00:16:28.437860520 +0100
+++ src/network.c	2017-11-17 00:25:57.623783100 +0100
@@ -21,6 +21,8 @@
 #include <string.h>
 #include <stdlib.h>
 
+#include <priv.h>
+
 void
 network_accept_tcp_nagle_disable (const int fd)
 {
@@ -387,6 +389,8 @@
 
 int network_init(server *srv, int stdin_fd) {
 	size_t i;
+	priv_set_t *tset;
+
 	if (0 != network_write_init(srv)) return -1;
 
 	{
@@ -425,6 +429,16 @@
 			if (0 != network_server_init(srv, dc->string, i, -1)) return -1;
 	}
 
+	/* here we drop privileges we won't need any more */
+	tset = priv_allocset();
+	priv_emptyset(tset);
+	priv_addset(tset, PRIV_NET_PRIVADDR);
+	if (setppriv(PRIV_OFF, PRIV_PERMITTED, tset) != 0) {
+		perror("Unable to set privileges: ");
+		return -1;
+	}
+
+
 	return 0;
 }
 
