--- llvm-3.3.src/include/llvm/Support/Host.h	Mon Apr 15 22:13:59 2013
+++ llvm-3.3.src/include/llvm/Support/Host.h	Sun Sep  1 22:06:36 2013
@@ -18,6 +18,12 @@
 
 #if defined(__linux__)
 #include <endian.h>
+#elif defined (__sun__)
+#include <sys/isa_defs.h>
+#if defined (_BIG_ENDIAN)
+#define BIG_ENDIAN 1
+#define BYTE_ORDER BIG_ENDIAN
+#endif
 #else
 #ifndef LLVM_ON_WIN32
 #include <machine/endian.h>
