--- configure.orig	Sa. Okt  8 21:40:50 2016
+++ configure	Mi. Okt 26 22:31:32 2016
@@ -12582,7 +12582,7 @@
     LIBS="$mutt_save_LIBS"
 
     if test "$with_idn" != auto; then
-      if test $have_stringprep_h = no || test $have_idna_h = no || test $ac_cv_search_stringprep_check_version = no; then
+      if test "$have_stringprep_h" = no || test "$have_idna_h" = no || test "$ac_cv_search_stringprep_check_version" = no; then
 	as_fn_error $? "IDN was requested, but libidn was not usable on this system" "$LINENO" 5
       fi
     fi
