--- old/readconf.c.orig	2014-07-18 06:11:26.000000000 +0200
+++ openssh-6.7p1/readconf.c	2015-02-24 16:30:51.432877189 +0100
@@ -266,7 +266,26 @@
 	{ "streamlocalbindmask", oStreamLocalBindMask },
 	{ "streamlocalbindunlink", oStreamLocalBindUnlink },
 	{ "ignoreunknown", oIgnoreUnknown },
-
+#ifdef DEPRECATE_SUNSSH_OPT
+	/* 
+	 * On Solaris, to make the transition from SunSSH to OpenSSH as smooth
+	 * as possible, we will deprecate SunSSH-only options in OpenSSH. 
+	 * Therefore, on a system that is running OpenSSH with a deprecated
+	 * option from the user's config file (~/.ssh/config), the ssh
+	 * connection will proceed without the deprecated option. Note that
+	 * this is an interim enhancement to OpenSSH to make the transition
+	 * smoother.  If a deprecated SunSSH-only option is migrated to OpenSSH
+	 * later, then it will be changed from deprecated to supported.
+	 */
+	{ "disablebanner", oDeprecated },
+	{ "gssapikeyexchange", oDeprecated },
+	{ "kmfpolicydatabase", oDeprecated },
+	{ "kmfpolicyname", oDeprecated },
+	{ "trustedanchorkeystore", oDeprecated },
+	{ "usefips140", oDeprecated },
+	{ "useopensslengine", oDeprecated },
+	{ "maxauthtrieslog", oDeprecated },
+#endif
 	{ NULL, oBadOption }
 };
 
