Calling "git repo-config" produces a "deprecated" warning

--- src/git.cpp.orig	Thu Mar 12 13:14:31 2009
+++ src/git.cpp	Sat Dec  5 01:38:20 2015
@@ -425,7 +425,7 @@
 	if (name == "Big5-HKSCS")
 		name = "Big5";
 
-	run("git repo-config i18n.commitencoding " + name);
+	run("git config i18n.commitencoding " + name);
 }
 
 QTextCodec* Git::getTextCodec(bool* isGitArchive) {
@@ -435,7 +435,7 @@
 		return NULL;
 
 	QString runOutput;
-	if (!run("git repo-config --get i18n.commitencoding", &runOutput))
+	if (!run("git config --get i18n.commitencoding", &runOutput))
 		return NULL;
 
 	if (runOutput.isEmpty()) // git docs says default is utf-8
