--- simgear-2.10.0/simgear/scene/sky/cloud.cxx.orig	2013-02-25 17:06:56.819955193 +0000
+++ simgear-2.10.0/simgear/scene/sky/cloud.cxx	2013-02-25 17:09:45.211593555 +0000
@@ -736,7 +736,7 @@
             // this happens, lets just use the last known good course.
             // This is a hack, and it would probably be better to make
             // calc_gc_course_dist() more robust.
-            if ( isnan(course) ) {
+            if ( std::isnan(course) ) {
                 course = last_course;
             } else {
                 last_course = course;
--- simgear-2.10.0/simgear/sound/sample_group.cxx.orig	2013-02-25 17:07:44.104263632 +0000
+++ simgear-2.10.0/simgear/sound/sample_group.cxx	2013-02-25 17:13:02.394452967 +0000
@@ -24,6 +24,7 @@
 #  include <simgear_config.h>
 #endif
 
+#include <cmath>
 #include <cassert>
 #include <simgear/compiler.h>
 #include <simgear/sg_inlines.h>
@@ -36,7 +37,7 @@
 using std::string;
 
 bool isNaN(float *v) {
-   return (isnan(v[0]) || isnan(v[1]) || isnan(v[2]));
+   return (std::isnan(v[0]) || std::isnan(v[1]) || std::isnan(v[2]));
 }
 
 SGSampleGroup::SGSampleGroup () :
