--- simgear-2.10.0/simgear/misc/sg_dir.cxx.orig	2013-02-18 14:58:14.000000000 +0000
+++ simgear-2.10.0/simgear/misc/sg_dir.cxx	2013-02-25 17:26:06.224950313 +0000
@@ -41,6 +41,7 @@
 #include <simgear/debug/logstream.hxx>
 #include <boost/foreach.hpp>
 
+#include <cstdlib>
 #include <cstring>
 #include <cstdlib>
 #include <iostream>
--- simgear-2.10.0/simgear/props/condition.cxx.orig	2013-02-25 17:24:58.437051448 +0000
+++ simgear-2.10.0/simgear/props/condition.cxx	2013-02-25 17:26:06.227381844 +0000
@@ -12,6 +12,7 @@
 #endif
 
 // #include <iostream>
+#include <cstdlib>
 
 #include <simgear/structure/exception.hxx>
 
--- simgear-2.10.0/simgear/props/propertyObject_test.cxx.orig	2013-02-18 14:58:14.000000000 +0000
+++ simgear-2.10.0/simgear/props/propertyObject_test.cxx	2013-02-25 17:26:06.230896178 +0000
@@ -14,6 +14,7 @@
 #include <cassert>
 #include <cstdlib>
 #include <cstring>
+#include <cstdlib>
 
 #include "propertyObject.hxx"
 
