--- vlc-2.0.3/modules/audio_output/oss.c.orig	2011-12-08 19:00:26.000000000 +0100
+++ vlc-2.0.3/modules/audio_output/oss.c	2012-08-04 11:42:36.377977552 +0200
@@ -34,6 +34,7 @@
 #include <fcntl.h>                                       /* open(), O_WRONLY */
 #include <sys/ioctl.h>                                            /* ioctl() */
 #include <unistd.h>                                      /* write(), close() */
+#include <sys/file.h>
 
 #include <vlc_common.h>
 #include <vlc_plugin.h>
